docs/mainComponent/LICENSE

```diff
@@ -0,0 +1,373 @@
+Mozilla Public License Version 2.0
```

Comment:
Can we get rid of this?

Reply:
Will do some research on this and see if its needed